### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sree94**
:)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# My-first
\ No newline at end of file
+# My-first
+Hi all,
```
Comment (sree94) [inline README.md]: from azureml import Workspace
import pandas as pd
import numpy as np
import matplotlib as plt
import sklearn

ws = Workspace()
ds = ws.datasets['train (2).csv']
train = ds.to_dataframe()


ds1 = ws.datasets['test (1).csv']
test = ds1.to_dataframe()
test_modified=ds1.to_dataframe()

a=test.isnull().sum()
#print(a)
mean_age=np.mean(train.Age)
train.Age = train.Age.fillna(mean_age)

mean_age1=np.mean(test_modified.Age)
test_modified.Age = test_modified.Age.fillna(mean_age1)
#print(test_modified)

train = train.drop(['Cabin','Ticket'],axis=1)
train['Age'] = np.log(train['Age'])
#print(train)

from sklearn.preprocessing import LabelEncoder
number = LabelEncoder()
train['Sex'] = number.fit_transform(train['Sex'].astype(str))
#print(train['Sex_New'])
#print(train)

test_modified['Sex'] = number.fit_transform(test_modified['Sex'].astype(str))
test_modified['Age'] = np.log(test_modified['Age'])
#print(test_modified)

import sklearn.linear_model
predictors =['Sex','Age','Pclass']
x_train = train[predictors].values
#print(x_train)
y_train = train['Survived'].values

#print(x_test)

model = sklearn.linear_model.LogisticRegression()
model.fit(x_train,y_train)
x_test = test_modified[predictors].values
#print(x_test)
